src/Proxy.cpp

```diff
@@ -167,6 +167,16 @@ void Proxy::registerSignalHandler( const std::string& interfaceName
     SDBUS_THROW_ERROR_IF(!inserted, "Failed to register signal handler: handler already exists", EINVAL);
 }
 
+void Proxy::unregisterSignalHandler( const std::string& interfaceName
+                                   , const std::string& signalName )
+{
+    auto& interface = interfaces_[interfaceName];
```

Comment:
Recommend using `at()` or `find()` here, since `unregisterSignalHandler()` should never add a new entry to `interfaces_`.

Reply:
Good point. I've created a follow-up MR: https://github.com/Kistler-Group/sdbus-cpp/pull/226